storefront-reference-architecture-master/cartridges/app_storefront_base/cartridge/templates/default/cart/cart.isml

```diff
@@ -63,7 +64,13 @@
                                 <isinclude template="cart/productCard/cartProductCard" />
                             </isif>
                         </isif>
+                    
                     </isloop>
+                
+                    <isif condition="${pdict.items.length>1}">
```

Comment:
Please watch out for the white spaces and indentations.

Reply:
The white space is removed.